inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/ProxyClientConfig.java

```diff
@@ -103,20 +103,20 @@ public class ProxyClientConfig {
 
     /* pay attention to the last url parameter ip */
     public ProxyClientConfig(String localHost, boolean isLocalVisit, String managerIp,
-            int managerPort, String groupId, String netTag, String authSecretId, String authSecretKey,
+            int managerPort, String inlongGroupId, String netTag, String authSecretId, String authSecretKey,
             LoadBalance loadBalance, int virtualNode, int maxRetry) throws ProxysdkException {
         if (Utils.isBlank(localHost)) {
             throw new ProxysdkException("localHost is blank!");
         }
         if (Utils.isBlank(managerIp)) {
             throw new IllegalArgumentException("managerIp is Blank!");
         }
-        if (Utils.isBlank(groupId)) {
+        if (Utils.isBlank(inlongGroupId)) {
             throw new ProxysdkException("groupId is blank!");
         }
         this.proxyIPServiceURL =
-                "http://" + managerIp + ":" + managerPort + ConfigConstants.MANAGER_DATAPROXY_API + groupId;
-        this.groupId = groupId;
+                "http://" + managerIp + ":" + managerPort + ConfigConstants.MANAGER_DATAPROXY_API + inlongGroupId;
```

Comment:
http is written hard in the url. It is recommended to make it configurable because some environments need to use https, such as the external network.